agrirouter-api-dotnet-standard-impl/Service/Messaging/Abstraction/SendMessageBaseService.cs

```diff
@@ -38,7 +34,7 @@ public MessagingResult Send(SendMessageParameters sendMessageParameters)
 
             if (string.IsNullOrWhiteSpace(sendMessageParameters.Base64MessageContent))
             {
-                throw new CouldNotSendMessageException(HttpStatusCode.BadRequest,
+                throw new CouldNotSendHttpMessageException(HttpStatusCode.BadRequest,
```

Comment:
That's not really an Http-Error, could better be a neutral CouldNotSendMessageException that is not Protocol specific